third_party/outline-ss-server/download.sh

```diff
@@ -18,8 +18,7 @@
 # Requires VERSION, OUTPUT and BASENAME to be defined.
 
 declare -r ARCHIVE="${BASENAME}.tar.gz"
-# We use wget instead of curl because it's already available on Alpine distros.
```

Comment:
I think it's worthy to review this comment:

> We use wget instead of curl because it's already available on Alpine distros.

What would happen to Alpine distros after the change?

Reply:
File is now deleted